packages/gtmcore/gtmcore/inventory/inventory.py

```
@@ -57,6 +60,12 @@ def __eq__(self, other) -> bool:
         return isinstance(other, InventoryManager) \
                and self.inventory_root == other.inventory_root
 
+    @staticmethod
+    def _windows_remove_readonly(func, path, excinfo):
+        """Error handler to remove readonly state on files during rmtree ops in windows"""
```

Took me a minute to figure out this is because sometimes artifacts are left after the delete b/c readonly files cannot be deleted, right?

---

That's the hypothesis. Pretty sure that when you import, there is something in the .git dir that is read-only on the windows file system. This then causes issues when `shutil.rmtree` runs. We were ignoring errors, so it was leaving the directory behind. On the next import it would fail.